ducktape/mark/resource.py

```diff
@@ -70,6 +70,8 @@ def the_test(x):
 
     """
 
+    # TODO: should I change this to also support node_spec in addition to num_nodes?
```

Comment:
I think we would have to for the test runner to work properly. Without that support, it can't schedule new tests in parallel since it'll need to take into account which tests need a windows box and how many are available.